apps/condo/domains/property/schema/RegisterResidentService.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generated by `createservice property.RegisterResidentService --type mutations`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/property/access/RegisterResidentService')
+const { Resident } = require('../utils/serverSchema/index')
+
+
+const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterResidentInput { dv: Int!, sender: JSON!, address: String!, addressMeta: JSON!, unitName: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canRegisterResident,
+            schema: 'registerResident(data: RegisterResidentInput!): Resident',
```

Comment:
useless service

Reply:
After business-requirements will be finalised, this service will do the following:
1. Connect `Resident` to a `Property` of verified `Organization`
2. Create a `Contact` for verified `Organization`.